packages/client/src/components/interface/Navigation.tsx

```diff
@@ -553,7 +553,7 @@ const NavigationView = (props: IFullProps) => {
                       }}
                     />
                   )}
-                {window.config.EXTERNAL_VALIDATION_WORKQUEUE && (
+                {window.config.FEATURES?.EXTERNAL_VALIDATION_WORKQUEUE && (
```

Comment:
Is the `FEATURES:` key required or optional?

Reply:
it is required indeed. It was a temporary change, I am going to correct it